src/any_agent/config.py

```diff
@@ -56,6 +56,8 @@ class MCPStdio(BaseModel):
     If none is specified, the default behavior is that the agent will have access to all tools under that MCP server.
     """
 
+    cwd: str | None = None
```

Comment:
Please add a description for this parameter, similar to how it's done with the others.